### PR TITLE
Install cryptography with correct version

### PR DESCRIPTION
**Pull request by lyw07**
## Summary

Previously the cryptography version given in the requirement.txt was not used for download in `install_cexts.py`. This caused an error that pip downloads the latest version of cryptography, while `pip install` was still trying to find the given version, `2.0.3`, of cryptography to install. 
The error indicated in the log file is:
> Requirement 'kolibri/dist/cext/cp27/win-amd64/cryptography-2.0.3-cp27-cp27m-win_amd64.whl' looks like a filename, but the file does not exist

> Installing cryptography-2.0.3-cp35-cp35m-macosx_10_6_intel.whl...
Installation failed for package 

What's changed in this PR is to include the given version when downloading the specific wheel file we need and also the code to fail buildkite in case errors like this happen again in the future so we will be alerted. 

## Issues addressed
https://github.com/learningequality/kolibri/issues/2433

### CODE REVIEW

**Comment by lyw07** [comment]
Tested with Mac OS X/Ubuntu/Windows 10, the cryptography module is successfully imported.

**Comment by aronasorman** [comment]
Can you try the RPi as well @lyw07?

**Comment by lyw07** [comment]
@aronasorman tested with raspberry pi on Python 3 (since we only have python 3 wheel file), and it works there.

File: install_cexts.py [hunk 1]
```diff
@@ -56,20 +57,24 @@ def download_package(path, platform, version, implementation, abi, name):
...
     """
-    return_code = subprocess.call(['python', 'kolibripip.pex', 'install', '-q', '-t', 
-        path, os.path.join(path, name)])
+    return_code = subprocess.call([
+        'python', 'kolibripip.pex', 'install', '-q', '-t',
+        path, os.path.join(path, name)
+    ])
     if return_code == 1:
-        print ('Installation failed for package {}\n'.format(name))
+        sys.exit('\nInstallation failed for package {}.\n'.format(name))
```
Comment (benjaoming) [inline install_cexts.py:77]: Ah yes, seems critical! :)

**Pull request merged**